### ISSUE TITLE
Cannot Edit PQ Expressions for Tables that have a Refresh Policy

### ISSUE THREAD

**Issue by tobimax** [TabularEditor/TabularEditor3#71]
Hi,

The PQ editor is really great, however you cannot edit PQ expressions for tables that have a Refresh Policy. It would be great if the 
SourceExpression and the PollingExpression properties were handled as per the standard single partition PQ editor.

Regards

Toby

**Comment by otykier** [comment]
I agree - this is an oversight. As a workaround, until it's fixed, you can technically still edit these expressions through the PropertyGrid, but I agree that it should work through the standard editor too.

**Comment by tobimax** [comment]
Yes, it's not a biggie at the moment. Thanks for logging it.
